cmd/geth/config.go

```diff
@@ -54,6 +55,9 @@ var (
 	}
 )
 
+var empty = struct{}{}
+var deprecatedConfig map[string]struct{} = map[string]struct{}{}
```

Comment:
Another way where it doesn't just clutter the global scope would be to encapsulate in a function: 
```golang
func deprecated(field string) bool{
	switch(field){
		case "foobar" , "bazonk": return true
	}
	return false
}
```

Reply:
Fixed, thanks